### ISSUE TITLE
Load function message

### ISSUE THREAD

**Issue by jeremy-keenan** [acids-ircam/nn_tilde#45]
A load function message for nn~ would be great, so different models could be loaded programmatically.

**Comment by caillonantoine** [comment]
I agree, however this cannot be done since each model might have a different number of inlets / outlets. What you can do is build abstractions with `nn~ #1` and pass the model name as an argument, then instantiate all abstractions and dynamically enabling / disabling the one you need !

**Comment by jeremy-keenan** [comment]
Ah, yes that could work well in most situations thank you.

What I was actually wanting to do is have a periodically updating model that would reload the newest model once it was done training automatically, but I can see the difficulty there.

I wonder if such a thing could be possible with mc/mcs.nn~? Or just with the 'forward' argument?

**Comment by caillonantoine** [comment]
What we could do is a "reload" method though... I'll se what I can do

**Comment by jeremy-keenan** [comment]
Thank you Antoine. Together with the already existing 'enable' method, it would be very flexible.

**Comment by jeremy-keenan** [comment]
Fantastic, thank you Antoine! I will be using this soon in an active project and will report back.
